### ISSUE TITLE
Clinical data question regarding tumor information

### ISSUE THREAD

**Issue by YeechingTiger** [neheller/kits19#64]
Hello @neheller,

Thanks for providing this excellent dataset. I have a question about the tumor information reported in the clinical data.

Regarding the situation that there are multiple masses in the CT scans, which is the tumor reported (e.g., malignant, pathologic size, etc) in the clinical data? Is it the largest mass?

**Comment by neheller** [comment]
Thanks for the great question.

This is an unfortunate limitation of the data in it's current form. I believe there are 12 multifocal cases in the dataset (6 ipsilateral, 6 bilateral) and radiologic attributes should describe only the _largest enhancing_ mass, and pathologic attributes describe the _largest excised_ specimen, but there could be exceptions to this.

We have recently been exploring whether it will be feasible to report attributes for each _mass_ rather than each patient, but since it's a retrospective cohort, this level of detail is not always available. 

We plan to release updated clinical data for these and more patients after the ongoing [KiTS21](https://github.com/neheller/kits21) Challenge, and I'll try to remember to come back and update this issue with how we decided to handle multifocal cases in that expanded cohort.

**Comment by neheller** [comment]
Apologies, didn't mean to close the issue.

**Comment by YeechingTiger** [comment]
Thanks!  Is it possible that you could release the clinical data for the last 90 cases  (i.e., case 210 - case 299)?

**Comment by szw1223** [comment]
> Apologies, didn't mean to close the issue.

Really thanks for providing this excellent dataset! Would you like to release the clinical data of the last 90 cases?

**Comment by szw1223** [comment]
Really thanks for providing this excellent dataset! Would you like to release the clinical data of the last 90 cases?

**Comment by neheller** [comment]
We are planning to at some point in the new two months, since we're hoping to wrap the KiTS19 test set into the [KiTS21](https://github.com/neheller/kits21) training set. Do you need it sooner? If so we could possibly make that work if you promise not to share it prematurely or use it to help with a KiTS19 submission :)

**Comment by YeechingTiger** [comment]
> We are planning to at some point in the new two months, since we're hoping to wrap the KiTS19 test set into the [KiTS21](https://github.com/neheller/kits21) training set. Do you need it sooner? If so we could possibly make that work if you promise not to share it prematurely or use it to help with a KiTS19 submission :)

Thanks!  We are not focusing on the segmentation task and we are more interested in the classification of some clinical variables using this excellent dataset.  But we encountered data imbalance issues and we are looking for more supplement data sources.  **We promise not to share it prematurely or use it to help with a KiTS19 submission.**

**Comment by neheller** [comment]
Good enough for me! Shoot me an email at helle246@umn.edu and I'll respond with the data.
